Cargo.toml

```diff
@@ -23,6 +23,7 @@ record_location_unstable = []
 slog = "2.4"
 slog-scope = "4"
 log = { version = "0.4.11", features = ["std"] }
+cached = "0.23.0"
```

Comment:
Hmmm... do we need a feature flag for this? :thinking:

Reply:
Ah, there's already a feature flag, so you want to add it here as well.

Reply:
I forgot about optional dependencies. Good call